Bot-agent/bot/bot.go

```diff
@@ -0,0 +1,144 @@
+package bot
+
+import (
+	"fmt";
+	"time";
+)
+
+var dictWeekdaysRus = [...]string {
+	"воскресенье",
+	"понедельник",
+	"вторник",
+	"среда",
+	"четверг",
+	"пятница",
+	"суббота",
+}
+
+var dictMonthesRus = [...]string{
+	"dimmy",
+	"января",
+	"февраля",
+	"марта",
+	"апреля",
+	"мая",
+	"июня",
+	"июля",
+	"августа",
+	"сентября",
+	"октября",
+	"ноября",
+	"декабря",
+}
+
+type Boter interface {
+	Response(cmd string) (ExitFlag int)
+	SayHello() 
+	SayTime()
+	SayDate()
+	SayWeekDay()
+	SayBye()
+}
+
+type BotRus struct {}
+type BotEn struct {}
+
+func CreateBot(language string) Boter {
+	switch language {
+		case "0": return new(BotRus)
+		case "1": return new(BotEn)
+		default: return nil
+	}
+}
+
+/* *************************************************************
+ *			Русский бот
+ * ************************************************************* */
+func (bot BotRus) Response(cmd string) (ExitFlag int) {
+	switch cmd {
+	case "1","Привет": bot.SayHello()
+	case "2","Время": bot.SayTime()
+	case "3","Дата": bot.SayDate()
+	case "4","День": bot.SayWeekDay()
+	case "5","Пока": bot.SayBye(); return 1
+	default: fmt.Println("Бананов нямя, попробуй что-нибудь еще")
+	}
+
+	return 0
+}
+
+func (bot BotRus) SayHello() {
```

Comment:
all bot commands must return a string because we will not able to use this bots in any app without a console.
output to the user should be at the UI layer  (main.go in our case)